Configuration/Geometry/python/GeometryExtended2023D5Reco_cff.py

```diff
@@ -45,6 +45,10 @@
 from Geometry.CSCGeometryBuilder.idealForDigiCscGeometry_cff import *
 from Geometry.DTGeometryBuilder.idealForDigiDtGeometry_cff import *
 
+#timing
```

Comment:
@lgray - ok, I think, we should auto-generate these files by default to avoid miss-synchronization (not a high priority though). I'll discuss it with @smuzaffar